docs/esm_runscripts.rst

```diff
@@ -2,6 +2,37 @@
 ESM Runscripts
 ==============
 
+Usage
+-----
+
+::
+
+    esm_runscripts [-h] [-d] [-v] [-e EXPID] [-c] [-P] [-j LAST_JOBTYPE]
+                      [-t TASK] [-p PID] [-x EXCLUDE] [-o ONLY]
+                      [-r RESUME_FROM] [-U]
+                      runscript
+
+Arguments
+---------
+
+====================================================== ==========================================================
+Optional arguments                                     Description
+====================================================== ==========================================================
+  -h, --help                                           Show this help message and exit.
+  -d, --debug                                          Print lots of debugging statements.
+  -v, --verbose                                        Be verbose.
+  -e ``EXPID``, --expid ``EXPID``                      The experiment ID to use. Default ``test``.
+  -c, --check                                          Run in check mode (don't submit job to supercomputer).
+  -P, --profile                                        Write profiling information (esm-tools).
+  -j ``LAST_JOBTYPE``, --last_jobtype ``LAST_JOBTYPE`` Write the jobtype this run was called from (esm-tools internal).
+  -t ``TASK``, --task ``TASK``                         The task to run. Choose from: ``compute``, ``post``, ``couple``, ``tidy_and_resubmit``.
+  -p ``PID``, --pid ``PID``                            The PID of the task to observe.
+  -x ``EXCLUDE``, --exclude ``EXCLUDE``                E[x]clude this step.
+  -o ``ONLY``, --only ``ONLY``                         [o]nly do this step.
+  -r ``RESUME_FROM``, --resume-from ``RESUME_FROM``    [r]esume from this step.
+  -U, --update                                         [U]date the tools from the current version.
```

Comment:
With the new feature in esm-tools/esm_runscripts#41 `-U` changes it functionality:

Before this feature the functionality of `-U` was just to copy `esm_tools` and default namelists into the experiment folder, but none of those were actually used. The esm_tools used are currently always the ones from the installation as far as I understand.

After the above mentioned feature `-U` updates the runscripts and associated files (through `further_reading`) in the experiment directory, so I would change [U] to be:

`[U]pdate the runscript in the experiment folder and associated files.`

Reply:
Ok. I will change this.

Thias is the explanation you get when calling `esm_runscripts --help`
This needs to be updated also.

Reply:
That's a good point! I will change that in esm-tools/esm_runscripts#41

Reply:
Are the steps in -x, -o-, -r the same as task in -t ?

Reply:
I've never used these flags :/

Reply:
Me neither. I leave it for now and merge it. May be it is clear anyway. But if it is the same you could name it the same.